src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int NUMBER_OF_FIGURE = 5;
```

Comment:
Let's make it private because you don't use it outside this class

Reply:
done